src/renderer/coremods/recovery/styles.css

```diff
@@ -0,0 +1,21 @@
+[class*="errorPage"] [class*="buttons_"] {
+  flex-direction: column;
+  align-items: center;
+}
+[class*="errorPage"] [class*="buttons_"] [class*="button__"] {
```

Comment:
This breaks all types of other buttons on the site. It may work intentionally for what you want, but it breaks the other classes that have them. For example, connect to a VC and you'll see.

Reply:
I'm not seeing it. You sure you are on the updated code?

Reply:
Fixed